charts/kf-notebooks/values.yaml

```diff
@@ -0,0 +1,52 @@
+# Global settings
+global:
+  storageClassName: ""
+
+# Common settings
+commonLabels: {}
+commonAnnotations: {}
+nameOverride: ""
+fullnameOverride: ""
+
+# Service settings
+service:
+  enabled: true
+  port: 80
+  type: ClusterIP
+
+# Notebook settings
+notebooks:
+  pytorch:
```

Comment:
Instead of pytorch we can say jupyter. Since this would cover more than pytorch.